pages/pipelines/group_step.md.erb

```diff
@@ -0,0 +1,213 @@
+# Group Step
+ 
+A *group* step groups jobs and other steps into logical groups, and allows displaying them within groups on the build page in Buildkite UI.    
+
+For example, your build pipelines in Buildkite are able to support an unlimited number of jobs. However, a large number of jobs on the build page can look messy. If pipelines contain 4-5 steps that can be logically grouped together, you can put them in a group using the `group` step.  
```

Comment:
This isn't really an example

> For example, your build pipelines in Buildkite are able to support an unlimited number of jobs.

Reply:
And 4-5 is kinda random. Does it makes more sense to say you can group all of your linting steps, and all of your UI test steps, for example?